tr/refresher-linear-algebra.md

```diff
@@ -1,339 +1,339 @@
 **1. Linear Algebra and Calculus refresher**
 
-&#10230;
+&#10230; Doğrusal Cebir ve Kalkülüs hatırlatması
 
 <br>
 
 **2. General notations**
 
-&#10230;
+&#10230; Genel notasyonlar
 
 <br>
 
 **3. Definitions**
 
-&#10230;
+&#10230; Tanımlar
 
 <br>
 
 **4. Vector ― We note x∈Rn a vector with n entries, where xi∈R is the ith entry:**
 
-&#10230;
+&#10230; Vektör - i-inci elemanı xi∈R olmak üzere n elemanlı bir vektör, x∈Rn:
 
 <br>
 
 **5. Matrix ― We note A∈Rm×n a matrix with m rows and n columns, where Ai,j∈R is the entry located in the ith row and jth column:**
 
-&#10230;
+&#10230; Matris ― Ai,j∈R i-inci satır ve j-inci sütundaki elemanları olmak üzere m satırlı ve n sütunlu bir matris, A∈Rm×n:
 
 <br>
 
 **6. Remark: the vector x defined above can be viewed as a n×1 matrix and is more particularly called a column-vector.**
 
-&#10230;
+&#10230; Uyarı: Yukarıda tanımlanan x vektörü n×1 tipinde bir matris olarak ele alınabilir ve genellikle sütun vektörü olarak adlandırılır.
 
 <br>
 
 **7. Main matrices**
 
-&#10230;
+&#10230; Ana matrisler
 
 <br>
 
 **8. Identity matrix ― The identity matrix I∈Rn×n is a square matrix with ones in its diagonal and zero everywhere else:**
 
-&#10230;
+&#10230; Birim matris ― Birim matris, köşegeni birlerden ve diğer tüm elemanları sıfırlardan oluşan karesel matris, I∈Rn×n:
 
 <br>
 
 **9. Remark: for all matrices A∈Rn×n, we have A×I=I×A=A.**
 
-&#10230;
+&#10230; Uyarı: Her A∈Rn×n matrisi için A×I=I×A=A eşitliği sağlanır.
```

Comment:
I think "Dipnot" or "Hatırlatma" is more appropriate translation for "Remark" here